terraform/lambda.tf

```
@@ -0,0 +1,117 @@
+resource "aws_lambda_function" "eks_scaling" {
+  function_name = "eks_scaling"
+  description   = "Upscale or downscale EKS cluster"
+  handler       = "lambda_function.lambda_handler"
```

```suggestion
  handler       = "eks_scaling.lambda_handler"
```

handler must be named after module and handler function